partition/bootloader_uboot.go

```diff
@@ -165,6 +165,10 @@ func (u *uboot) GetBootVar(name string) (value string, err error) {
 	return getBootVar(name)
 }
 
+func (u *uboot) SetBootVar(name, value string) error {
+	return setBootVar(name, value)
```

Comment:
The changes in this branch itself feel fine, but the two-layered indirection that is already there (SetBootVar => the global => the actual implementation) looks a bit curious. It'd be nice to move this to become simply a struct field on uboot telling whether the legacy implementation is in use or not, and then we can have the real implementation in here, keeping just the alternative function (e.g. {get,set}BootVarLegacy) and dropping the globals.

Either way, not for this branch. Thanks for listening. :)